test/test_lib.py

```diff
@@ -152,6 +152,18 @@ def test_install(run_command):
     assert "Error resolving dependencies for MD_Parola@3.2.0: dependency 'MD_MAX72xx' is not available" in result.stderr
 
 
+def test_install_flags(run_command):
```

Comment:
Could you write a test for the zip installation too?

Reply:
Yeah sure, I just figured I can `wget` a zipped library and test the install.

Reply:
Best to use the `lib download` command and then install that downloaded one, also I don't think `wget` is available on Windows.

To find the downloaded library you'll probably need the `downloads_dir` fixture for that test.